Gruntfile.js

```diff
@@ -289,6 +295,8 @@ module.exports = (grunt) => {
     grunt.log.writeln('backend: launching...')
     // Provides Babel support for the backend files.
     require('@babel/register')
+    // Make `GI_VERSION` available via `process.env` before loading our server.
+    process.env.GI_VERSION = GI_VERSION
```

Comment:
This should probably be set up at the top of this file.

Reply:
That's what I did initially. Then I thought it would be cleaner to not mutate `process.env` if possible.

Now since it's still going to be mutated in at least one case, one could argue that it would be better to be clear about it upfront and just put that mutation at the top of the file.

However, this looked a bit ugly. And `backend:launch` is the only place where the environment mutation is necessary, because of a local optimization to avoid creating a child process when launching the backend for the first time.

So I ended up moving this line here, but I can undo that if you think it's important.